images/docker-dind/Dockerfile

```diff
@@ -41,3 +46,10 @@ RUN tmpdir=`mktemp -d` && curl "https://awscli.amazonaws.com/awscli-exe-linux-x8
 	unzip $tmpdir/awscliv2.zip -d $tmpdir && \
 	$tmpdir/aws/install -i /usr/local/aws-cli -b /usr/local/bin && \
 	rm -r $tmpdir
+
+RUN apt-get update && TMPDIR=$(mktemp -d) \
```

Comment:
I can recommend you a better way of installing cosign into an image (optional):

> https://github.com/sigstore/cosign#dockerfile

Reply:
Good idea multi-stage here! thanks @developer-guy

Reply:
Applied, thanks!